docs/config.md

```diff
@@ -318,6 +319,100 @@ location /blog/ {
 
 Default: `[]`
 
+- `NoDNSLink`  
+  A boolean to configure whether DNSLink lookup for value in `Host`
+  HTTP header should be performed.  If DNSLink is present, content path stored
+  in the DNS TXT record is mounted at `/` and respective payload is returned to
+  the client.
+
+Default: `false`
+
+### Gateway.PublicGateways
+
+`PublicGateways` is a dictionary for defining gateway behavior on specific fully qualified domain names (FQDN).  
+Options for changing behavior per hostname:
+
+- `Paths`  
+  Array of paths that should be mounted at the root of domain name.  
+  Example: `["/ipfs", "/ipns", "/api"]`  
+  - **Note:** when both `Paths` and DNSLink are mounted, `Paths` takes priority
+
+- `UseSubdomains`
+  A boolean to configure whether the gateway provides [Origin isolation](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy)
+  between content roots.  
+  Default: `false`
+  - `true` – mount [subdomain gateway](#https://docs-beta.ipfs.io/how-to/address-ipfs-on-web/#subdomain-gateway) at `http://{cid}.ipfs.{hostname}`
+    - requires respective `Paths` to be set
```

Comment:
It does? I thought we only needed the path to be set for a redirect to happen. Also, are we using `http://api.hostname`? Or just `hostname/api`? I assume the latter.

Reply:
Yes, now that we are not reusing `PathPrefixes`, I want to make it more useful: 
- `Paths` define which content paths are handled by the subdomain gateway.
  - Example: `Paths: ["/ipfs"]` would enable `http://{immutable}.ipfs.{hostname}` but refuse `http://{mutable}.ipns.{hostname}`